recipes/server_setup.rb

```diff
@@ -92,6 +103,7 @@
       # Create option string
       force = false
       options = ''
+      force = true if node['gluster']['server']['disks'].empty?
```

Comment:
force is needed only when gluster is creating on root partition/volume (for example on kitchen ;). But i dont know how to easy check if there is that case. I think about conditional "force" option for volume configuration.